hibike/lib/hibike/hibike_device.cpp

```diff
@@ -113,12 +112,3 @@ void toggleLED() {
     led_enabled = true;
   }
 }
-
-uint64_t getLastHeartbeatResponse(){
```

Comment:
TY for removing these functions.

Instead, could we please have a function, like the following:

uint32_t get_time_since_heartbeat(){
return millis() - resp_heartbeat;
}

It might be helpful to have this function lying around for the future.